velox/exec/tests/SpillTest.cpp

```diff
@@ -493,3 +493,43 @@ TEST_F(SpillTest, spillPartitionSet) {
     }
   }
 }
+
+TEST_F(SpillTest, spillPartitionSpilt) {
+  for (int iter = 0; iter < 5; ++iter) {
+    int numBatches = 100;
+    std::vector<RowVectorPtr> batches;
+    batches.reserve(numBatches);
+
+    const int numRowsPerBatch = 100;
+    setupSpillState(
+        iter % 2 ? 1 : 1'000'000'000, 1, numBatches, numRowsPerBatch);
+    const SpillPartitionId id(0, 0);
+    ASSERT_TRUE(id.valid());
+
+    auto spillPartition =
+        std::make_unique<SpillPartition>(id, state_->files(0));
+    std::copy(
+        batchesByPartition_[0].begin(),
+        batchesByPartition_[0].end(),
+        std::back_inserter(batches));
+
+    folly::Random::DefaultGenerator rng;
```

Comment:
Do we need this randomness?

Reply:
I want to have different number of files versus number of splits here.